microcosm_flask/conventions/crud.py

```diff
@@ -54,9 +54,9 @@ def configure_search(self, ns, definition):
         @qs(definition.request_schema)
         @response(paginated_list_schema)
         def search(**path_data):
-            page = self.page_cls.from_query_string(definition.request_schema)
-            result = definition.func(**merge_data(path_data, page.to_dict(func=identity)))
-            response_data, headers = page.to_paginated_list(result, ns, Operation.Search)
+            in_page, out_page = self.page_cls.from_query_string(definition.request_schema)
```

Comment:
please see my comments in https://github.com/globality-corp/microcosm-flask/pull/130